server/src/constants.rs

```diff
@@ -6,11 +6,11 @@ pub const EXTENSION_NAME: &str = "Odoo";
 pub const EXTENSION_VERSION: &str = "1.0.1";
 
 pub const DEBUG_ODOO_BUILDER: bool = false;
-pub const DEBUG_MEMORY: bool = false;
+pub const DEBUG_MEMORY: bool = true;
 pub const DEBUG_THREADS: bool = false;
 pub const DEBUG_STEPS: bool = false;
 pub const DEBUG_STEPS_ONLY_INTERNAL: bool = true;
-pub const DEBUG_REBUILD_NOW: bool = false;
+pub const DEBUG_REBUILD_NOW: bool = true;
 pub const DEBUG_BORROW_GUARDS: bool = false;
```

Comment:
Same with those? do you want to have them enabled on Alpha? and we'll revert them on release?